tests/e2e/models/onboarding/ThemeSection.ts

```diff
@@ -0,0 +1,18 @@
+import { Page } from 'playwright';
+
+export class ThemeSection {
+	page: Page;
+
+	constructor( page: Page ) {
+		this.page = page;
+	}
+
+	async isDisplayed( ) {
+		await this.page.waitForSelector(
+			':text("Choose a theme")'
+		);
+        await this.page.waitForSelector(
```

Comment:
Something went awry with the indentation here. Do you have prettier enabled to auto format your ts files? I know Paul had to change a setting to make that happen.